app/uk/gov/hmrc/rasapi/services/RasfileReader.scala

```diff
@@ -47,7 +47,7 @@ trait RasFileReader {
     fileUploadConnector.getFile(envelopeId, fileId, userId).map{
 
       case Some(inputStream) => Source.fromInputStream(inputStream).getLines
-      case None => Logger.error(s"File Processing: problem reading data in the file for userId ($userId).")
+      case None => Logger.error(s"[RasFileReader][readFile] File Processing: problem reading data in the file for userId ($userId).")
```

Comment:
This error message doesn't' seem true? It looks like there was simply no file found rather than it being an error from being unable to read that file.